const.go

```diff
@@ -33,8 +33,10 @@ const (
 	// PrivDir is the directory, under the notary repo base directory, where private keys are stored
 	PrivDir = "private"
 	// RootKeysSubdir is the subdirectory under PrivDir where root private keys are stored
+	// The only reason we need this constant is compatibility with older versions
```

Comment:
Add `DEPRECATED` notice on front of this comment and the one for `NonRootKeysSubdir`